sdk/storage/storage-blob/src/AppendBlobClient.ts

```diff
@@ -5,8 +5,7 @@ import {
   HttpRequestBody,
```

Comment:
minor: please be aware of potential merges with other PRs, e.g. there are ongoing PR which merges code of AppendBlobClient.ts into BlobClient.ts.

Reply:
Might need to resolve merge conflicts.
(Please run all the tests before merging - `/azp run js - storage-blob - tests`)